coalib/output/Logging.py

```diff
@@ -1,28 +1,80 @@
 from datetime import datetime
-import json
 import io
+import json
 import logging
 import logging.config
 
+from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
```

Comment:
I wouldn't convert our old log level values to `logging`-module values. People that use your logging module shall use the new log levels :D